packages/relay-compiler/core/RelaySourceModuleParser.js

```diff
@@ -66,14 +66,18 @@ module.exports = (
 
     const astDefinitions = [];
     const sources = [];
-    memoizedTagFinder(text, baseDir, file).forEach(template => {
-      const source = new GraphQL.Source(template, file.relPath);
+    memoizedTagFinder(text, baseDir, file).forEach(tag => {
+      const source = new GraphQL.Source(
+        tag.template,
+        path.join(path.relative(process.cwd(), baseDir), file.relPath),
```

Comment:
Keep path short by making it relative, but in a way that the IDE can find it (i.e. relative to the working dir).

Reply:
@kassens This too.